installation.md

```diff
@@ -1,53 +1,85 @@
 # Installation
 
-Deployer can be installed in two ways: using phar archive and using composer.
+Deployer can be installed in three ways: 
+1) using **composer package with phar only** - later called also "Deployer Distribution" version (no dependencies)
+2) using **composer package with sources** - later called also "Deployer Source" version (dependencies but good for debug 
+   Deployer) 
+3) using direct download of **phar archive**
 
-### Phar archive
+The best option for most cases is to choose  **composer package with phar only**. 
 
-To install Deployer via phar archive, run the following commands:
+Two other methods have following disadvantages:
 
-```sh
-curl -LO https://deployer.org/deployer.phar
-mv deployer.phar /usr/local/bin/dep
-chmod +x /usr/local/bin/dep
-```
+* If you will install Deployer globally as **phar archive** then it will fit your needs well until the amount of 
+  projects will increase. Its quite possible that if you have big amount of projects then you will be not able to 
+  upgrade all deploy.php configs at each project at one time to newer version of deployer. So for some time you will 
+  need to have two or more versions of deployer installed globally under different names. For example "dep" for 
+  deployer5, "dep4" for deployer4 and then run commands like "dep4 deploy production" or "dep deploy production" - 
+  depends on which project you are. Additionally if new developer is coming into project he needs to install all 
+  versions of Deployer binaries on his local computer.
 
-If you need another version of Deployer, you can find it on the [download page](https://deployer.org/download).
-Later, to upgrade Deployer, run the command:
+* If you will install Deployer as **composer package with sources** then you risk that deployer dependencies 
```

Comment:
There is no risk, this is how composer and php works. When you install ANY of deps via composer you may endup in unresoved situation. Better omit this.